src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -457,7 +461,13 @@ If you have code that sets up global state, this state will need to take into ac
 </script>
 ```
 
-Module scripts, including those that add event listeners for `DOMContentLoaded`, are only ever executed once because the browser keeps track of which modules are already loaded. You must add listeners for a [Lifecycle event](#lifecycle-events) to re-execute scripts during client-side page navigation.
+Inline scripts can be re-executed after every transition by adding the `data-astro-rerun` property. This will make the script `is:inline` implicitly, so it will not work for bundled scripts.
+
+```astro
+<script is:inline data-astro-rerun>...</script>
+```
+
+If a script should execute every time a page is loaded, it should be executed by a [lifecycle event](#lifecycle-events). Event listeners for `DOMContentLoaded` can be replaced by the [`astro:page-load`](http://localhost:4321/en/guides/view-transitions/#astropage-load) lifecycle event.
```

Comment:
```suggestion
```
I suggest moving this earlier to where you're talking about whether or not scripts re-execute!